python/ray/util/dask/examples/dask_ray_shuffle_optimization.py

```diff
@@ -0,0 +1,29 @@
+import ray
+from ray.util.dask import dataframe_optimize
+import dask
+import dask.dataframe as dd
+import numpy as np
+import pandas as pd
+
+# Start Ray.
+# Tip: If connecting to an existing cluster, use ray.init(address="auto").
+ray.init()
+
+# Set the Dask DataFrame optimizer to
+# our custom optimization function, this time using the config setter as a
+# context manager.
+with dask.config.set(dataframe_optimize=dataframe_optimize):
```

Comment:
I had to remove `scheduler=dask_ray_get` from here to get this working with Ray Client. Is this expected? @rkooo567 @clarkzinzow

Reply:
@amogkam If `scheduler=ray_dask_get` is removed, this example won't use the Dask-on-Ray scheduler, it'll just run the Dask workload on a local thread pool and the optimization will be a no-op.

How does the example fail when the scheduler is set? That seems like a bug.